backend/cms/urls.py

```diff
@@ -173,6 +173,7 @@
                 url(r'^$', pages.PageTreeView.as_view(), name='pages'),
                 url(r'^new$', pages.PageView.as_view(), name='new_page'),
                 url(r'^upload$', pages.upload_page, name='upload_page'),
+                url(r'^upload$', pages.upload_page, name='upload_page'),
```

Comment:
Oh, this was just an unintended line duplication. Will remove it before I merge.